README.md

```diff
@@ -51,10 +55,12 @@ for all the passed options.
 ## Options
 
   - `cache`           Compiled functions are cached, requires `filename`
-  - `filename`        Used by `cache` to key caches, and for includes
+  - `filename`        The name of the file being rendered. Not required if you 
+  are using `renderFile()`. Used by `cache` to key caches, and for includes.
   - `context`         Function execution context
   - `compileDebug`    When `false` no debug instrumentation is compiled
-  - `client`          Returns standalone compiled function
+  - `client`          When `true`, compiles a function that can be rendered 
+    in the browser without depending on ejs.js.
```

Comment:
`ejs.js` sounds a bit ambiguous, since it can also refer to the project name. How about “EJS runtime”?